### PR TITLE
Fix: Add IP and ports to .env file

### PR DESCRIPTION
**Pull request by AngeloTadeucci**
### .env
- Split database info and added port;
- Added server IP, game and login port and server name to .env.

### Others
- Fixed `NpcTalkHandler` to not crash if NPC doesn't have script;
- Removed references of `Ms2Database` from `MapleServer2.sln`

### CODE REVIEW

**Review by AlanMorel — APPROVED** [review]

**Review by rhydev — CHANGES_REQUESTED** [review]
.env file keys should be in all caps, also DB_Database doesn't really make sense

**Review by rhydev — COMMENTED** [review]
Test comment

File: .env.example [hunk 1]
```diff
@@ -1 +1,12 @@
-DATABASE_URL:server=localhost;database=Maple2DB;user=root;password=root
\ No newline at end of file
+# Database Info
+DB_IP=localhost
+DB_PORT=3306
+DB_NAME=Maple2DB
+DB_USER=root
+DB_PASSWORD=root
+
```
Comment (AngeloTadeucci) [inline .env.example:7]: test

**Comment by rhydev** [comment]
Test

**Review by rhydev — APPROVED** [review]

**Pull request merged**